grazer/src/main/java/edu/asu/diging/grazer/web/LoginController.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.grazer.web;
+
+import java.security.Principal;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+public class LoginController {
+
+    @RequestMapping(value = "login", method = RequestMethod.GET)
+    public String validUserHandle(ModelMap model, Principal principal,
+            Authentication authentication) {
```

Comment:
The methods validUserHandle and login seem to have the same request mapping : /login